### PR TITLE
Connect to Anthropic directly

### PR DESCRIPTION
**Pull request by dooart**

### CODE REVIEW

**Comment by tobowers** [comment]
Should we do this here or in the engine? And in both cases, maybe it's a new processor?

**Comment by dooart** [comment]
@tobowers oh only saw your comment now. yeah i think we can easily move it to the engine

**Comment by dooart** [comment]
@tobowers what do you think of this approach? then we simply move the custom rest client to the engine

my goal here is to make it possible to just delete the custom client when we don't need it anymore

File: packages/core/package.json [hunk 1]
```diff
@@ -34,13 +34,14 @@
...
   "dependencies": {
-    "@anthropic-ai/sdk": "^0.19.0",
+    "@anthropic-ai/sdk": "^0.19.1",
     "@opentelemetry/api": "^1.8.0",
     "common-tags": "^1.8.2",
     "eventemitter3": "^5.0.1",
     "exponential-backoff": "^3.1.1",
     "gpt-tokenizer": "^2.1.2",
     "nanoid": "^5.0.6",
+    "node-fetch": "^3.3.2",
```
Comment (dooart) [inline packages/core/package.json:44]: by moving the custom client to the engine we can remove this too

**Pull request closed without merging**